File: 2025-11/2025-11-08/character_limit.py
def can_post(message):

    return "short post" if len(message) <= 40 else "long post" if len(message) <= 80 else "invalid post"

print(can_post("Hello world"))
print(can_post("This is a longer message but still under eighty characters."))
print(can_post("This message is too long to fit into either of the character limits for a social media post."))

File: 2025-11/2025-11-08/characterLimit.js
function canPost(message) {

  return message.length <= 40 ? "short post" : message.length <= 80 ? "long post" : "invalid post";
}

console.log(canPost("Hello world"));
console.log(canPost("This is a longer message but still under eighty characters."));
console.log(canPost("This message is too long to fit into either of the character limits for a social media post."));